_data/sites.json

```diff
@@ -4226,17 +4237,6 @@
         ]
     },
 
-    {
-        "name": "Invidious",
```

Comment:
What happened to this website? Doesn't exist anymore?

Reply:
commit message: invidio.us has closed down on September 1st.

Reply:
Yes, that's right.

Reply:
Oh, I had missed that, thank you all :)